src/libraries/Microsoft.Extensions.Http/tests/Logging/RedactedLogValueIntegrationTest.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Extensions.Http.Logging
 {
     public class RedactedLogValueIntegrationTest
     {
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
Some other files also don't have `using System;` at the top, like `src/libraries/System.Reflection.Metadata/tests/Utilities/AbstractMemoryBlockTests.cs` and `src/libraries/System.Collections.Concurrent/tests/ConcurrentDictionary/ConcurrentDictionaryTests.cs`. Does `RedactedLogValueIntegrationTest.cs` need it specifically because its namespace is not part of Systems?

Reply:
Yes, correct. You can see it in this example: https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxASwDYB8B2ExxQAOEAxnAAQCyapCA9lPQGYwB0AYvfRQN4CwAKAoiKAAQDM4gEwUAwnyGjl4qWIAs1ABQBKRcJWGxARgCcWgEQALOBgz0LOgNxLDFAL6uRnwT6F4CYjJKEzYaOkYWdi4eAQMRSRl5fTdVcU0qXRTU0RNza1t7Rxd4lR9lH3cgA